src/lib.rs

```diff
@@ -10,8 +10,11 @@
 //! let style = lscolors.style_for_path(path);
 //!
 //! // If you want to use `ansi_term`:
+//! # #[cfg(features = "ansi_term")]
+//! # {
```

Comment:
does this disable part of the docstring if `ansi_term` is not set? If so, should we include the "If you want to use `ansi_term`:" line as well?

Reply:
Ah no, that's required for doc tests :+1: